compiler/rustc_mir_transform/src/dataflow_const_prop.rs

```diff
@@ -599,13 +687,49 @@ impl<'tcx> Visitor<'tcx> for OperandCollector<'tcx, '_, '_, '_> {
 
 struct DummyMachine;
 
+/// Macro for machine-specific `InterpError` without allocation.
+/// (These will never be shown to the user, but they help diagnose ICEs.)
+macro_rules! throw_machine_stop_str {
```

Comment:
Instead of duplicating the macro, can you import it from the other pass / move it to a shared location?